interop/nkic/serde_python_core.h

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Govereau, Sean McLaughlin
+*/
+#pragma once
+
+// This file is automatically generated from KLR.
+// Manual edits to this file will be overwritten.
+
+#include "ast_python_core.h"
+#include "region.h"
+#include "stdc.h"
+
+bool Python_Pos_ser(FILE *out, struct Python_Pos *x);
```

Comment:
We should think about a larger error-reporting strategy. Not in this PR, but soon.

These functions could fail for multiple reasons: "error writing to file", "unknown tag value 1234", etc. If we just do `bool` to report errors, and something goes wrong deep in a stack of function calls, the cause will be mysterious to users and us.

Return an error code? An error struct? A Rust-like result type? Just do C++ and use exceptions? I'd personally vote that we have some kind of struct involved, not just an error code, so we can embed extra information like custom messages and line numbers. Or use C++ with exceptions.